### ISSUE TITLE
Disabled and Readonly not working in inline mode

### ISSUE THREAD

**Issue by vanling** [Vuepic/vue3-date-time-picker#70]
**Describe the bug**
`disabled=true` is not working when inline is set to true

**To Reproduce**
```
<Datepicker
    v-model="date"
    :disabled="true"
    :inline="true"
  />
  ```

**Expected behavior**
Disabled state, where you can not select a date

**temp fix**
`:class="[{ 'opacity-50 pointer-events-none': disabled }]"`

**Comment by vanling** [comment]
Maybe the same with "readonly" 
So disabled - everything gray like a disabled date, nothing clickable
and with readonly="true", nothing clickable but keep the styling?

**Comment by Jasenkoo** [comment]
Hmm, I didn't even think about it 😄 . Will fix it

**Comment by Jasenkoo** [comment]
Fixed in [v2.4.3](https://github.com/Vuepic/vue3-date-time-picker/releases/tag/v2.4.3)
